src/arrayMethodSort.js

```diff
@@ -5,7 +5,31 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let step;
+    let callback = compareFunction;
+
+    if (!compareFunction) {
+      callback = function(curr, prev) {
+        return String(curr) < String(prev) ? -1 : 1;
+      };
```

Comment:
let's rewrite it using the arrow function it would be much more readable
and you need to return `0` if curr === prev